<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddPatient</class>
 <widget class="QDialog" name="DialogAddPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавления пользователя</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Resources/logo.ico</normaloff>../../Resources/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_logo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_fio">
       <property name="placeholderText">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_login">
       <property name="placeholderText">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_password">
       <property name="placeholderText">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLineEdit" name="le_b_day">
       <property name="placeholderText">
        <string>Дата рождения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_passport_series">
       <property name="placeholderText">
        <string>Серия паспорта</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_passport_number">
       <property name="placeholderText">
        <string>Номер паспорта</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="le_ein">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_policy_number">
       <property name="placeholderText">
        <string>Номер страхового полиса</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbox_type_policy">
       <property name="placeholderText">
        <string>Тип страхового полиса</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="le_phone">
       <property name="placeholderText">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_email">
       <property name="placeholderText">
        <string>e-mail</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QComboBox" name="cbox_name_company">
       <property name="placeholderText">
        <string>Название страховой компании</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_req">
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
background-color: rgb(73,140,81);
}</string>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
